.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+  # # Maintain dependencies for Python
+  # - package-ecosystem: pip
+  #   directory: /
+  #   schedule:
+  #     interval: weekly
```

Comment:
Nice! We can probably include python dependencies too?

In cloudapi I set the schedule as follows, to run on Saturday (instead of default Monday):
```yml
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      day: "saturday"
      time: "12:00"
```